### DISCUSSION TITLE
Add construction cheat: Allow all paths as queue

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18370] [Feature requests] [answered]
I found out that you can easily turn your regular paths into queues when using sadret's scenery manager, but there's currently no real other way to turn your regular into queues, although this would be quite a helpful feature for people that want to make more coherent and realistic looking parks. I wanted to suggest a checkbox cheat that adds all other regular paths to the queue tab when the checkbox is checked.

**Comment by ocalhoun6** [comment]
Would doing this require additional API hooks in OpenRCT2's code?

If not, then this is a feature you'd want to request from whoever maintains the sadret's scenery manager plugin. (Presumably sadret?) Changing a plugin is something that would need to be done to the plugin, not something that can be done in OpenRCT2 itself.

**Reply by karst** [reply]
You can already do this in the game, it's just only available by using the replace tool in the scenery manager and there's currently no other way. Making this a cheat makes it actually doable in game to have a working concrete queue for example.

Example:
![image](https://user-images.githubusercontent.com/1516893/196312372-e01fcd9c-48af-43ce-ab13-7fdb8fdf6b3b.png)

**Reply by MASHzj** [reply]
isn't this possible via cheats now?

**Comment by karst** [comment, marked as answer]
This was introduced in #18744
